mmd_tools/converters/mesh_converter.py

```
@@ -0,0 +1,619 @@
+                    sphere_file_node = cmds.shadingNode(
+                        "file", asTexture=True, name=shader + "_sphere_texture"
+                    )
+                    maya_utils.set_attribute(
+                        sphere_file_node, "fileTextureName", full_sphere_path, "string"
+                    )
+                    try:
+                        cmds.connectAttr(
+                            sphere_file_node + ".outColor",
+                            shader + ".SphereTexture",
+                            force=True,
+                        )
```

Ruff (0.12.2)

Do not use bare `except`

(E722)